packages/gafl-webapp-service/src/pages/summary/contact-summary/contact-summary.njk

```diff
@@ -44,9 +44,9 @@
 
             {{ csrf() }}
         </form>
-
+        
         <p class="govuk-body-m">
-            <a class="govuk-link" href="{{ data.uri.licenceSummary }}">{{ mssgs.change_licence_details }}</a>
+            <a class="govuk-link" href="{{ data.uri.licenceSummary }}">{{ mssgs.change_licence_details_you if data.isLicenceForYou else change_licence_details_other }}</a>
```

Comment:
Have you tested that msgs.change_licence_details_other appears? It doesn't look like it will.
I'd say decision logic like this should really be in the getData function rather than the template - so that it can be tested.